docs/reference/modules/i18n.md

```diff
@@ -122,6 +125,56 @@ Same thing if we request `example.ca` with no locale prefix, we will be redirect
   </template>
 </AposCodeBlock>
 
+### `adminLocales` {#adminlocales}
+
+The `adminLocales` takes an array of objects. Each of the objects should have a label that is presented to the user in both the user manager editor, and in the user preferences menu. This menu can be accessed through the dropdown at the right hand side of the admin-bar menu. The objects should also have a `value` property that takes the key for a locale defined in the `locales` option, that should be associated with this selection.
```

Comment:
There is no requirement for adminLocales to also appear under locales. If we have translations for a locale, or you bring them at project level, you can specify it for adminLocales even if you have no intention of offering content in that locale. This is fairly common ("this site is in three non-English languages but we want the admin UI in business English").

Reply:
Changed.